### PR TITLE
Add dependencies for tools-pocolog_pybind

### PR DESCRIPTION
**Pull request by mstoelzle**
- Add pybind11 as external/pybind
- Pybind requires ninja-build
- Activate python3 installations deactivated in rock-core package set
- Add `tools/pocolog_pybind` component

@levingerdes I will remove the wrong commit authors with a squash merge at the end

### CODE REVIEW

**Comment by mstoelzle** [comment]
@levingerdes I tested it on a fresh docker image and it seems to be working

File: source.yml [hunk 1]
```diff
@@ -139,6 +139,8 @@ version_control:
     type: git
     url: https://github.com/rock-drivers/drivers-gnss_trimble.git
     branch: master
+  - external/pybind11:
+    github: pybind/pybind11
```
Comment (levingerdes) [inline source.yml:143]: Please use the same format here as we do for all the other packages. I know it's two extra lines, but you can then just click on/copy the URL if needed and you specify the branch explicitly.

**Review by levingerdes — CHANGES_REQUESTED** [review]
I'd just request one tiny change before merging. See my comment in `source.yml`.

File: source.yml (hunk 1 above)
Comment (mstoelzle) [reply source.yml]: Yes, I've changed that. Thanks a lot

**Review by levingerdes — APPROVED** [review]

**Pull request merged**